timesync/config/config.go

```diff
@@ -0,0 +1,51 @@
+package config
+
+import (
+	"time"
+
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+// ConfigValues specifies  default values for node config params.
+var (
+	ConfigValues     = DefaultConfig()
+	TimeConfigValues = ConfigValues.TimeConfig
```

Comment:
This isn't loaded. from config file. check the loading in main and make sure its applied to here. (because this is used globally)

Reply:
I think that now it's fixed, even when I can not test it very well.